packages/palette/src/elements/DonutChart/DonutChart.tsx

```diff
@@ -151,18 +147,14 @@ export const DonutChart: React.FC<DonutChartProps> = ({
     </svg>
   )
 
-  const ChartSVG = () => (
-    <>
-      {tooltip}
-      {labels}
-      {svg}
-    </>
-  )
-
   return (
     <ProvideMousePosition>
       <Box ref={wrapperRef as any} position="relative">
-        <ChartSVG />
+        <>
```

Comment:
idk why but with this change got rid of:

<img width="1128" alt="Screen Shot 2019-06-19 at 2 19 28 PM" src="https://user-images.githubusercontent.com/687513/59790793-b9ba6780-929e-11e9-921c-b9596ea32b55.png">